### PR TITLE
Documentation updated- added important note for `inlinebot.py` example

### PR DESCRIPTION
**Pull request by Aditya-Rajgor**
Added a simple note on `inlinebot.py` python file in the **examples** directory. 

I found that it is necessary to enable inline commands for your bot from **BotFather** from the documentation
here - https://core.telegram.org/bots/inline

I hope it serves some value to the community.

### CODE REVIEW

**Review by Bibo-Joshi — COMMENTED** [review]
Hi. Thanks for PRing! While we do point to this [in the readme](https://github.com/python-telegram-bot/python-telegram-bot/tree/master/examples#inlinebotpy) already, there's surely no harm in repeating it in the example.

File: examples/inlinebot.py [hunk 1]
```diff
@@ -3,6 +3,11 @@
 # This program is dedicated to the public domain under the CC0 license.
 
 """
+IMPORTANT - 
+To enable inline commands for your bot, you have to send `/setinline` command to `botfather`.
+Then select your bot and set placeholder for your bot.
+Read more here - https://core.telegram.org/bots/inline
```
[the remark this answers is not in the document]
Comment (Aditya-Rajgor) [reply examples/inlinebot.py:9]: True, just pointing out in the example would be sufficient

[the remark this answers is not in the document]
Comment (Bibo-Joshi) [reply examples/inlinebot.py:9]: would you like to rephrase the note?

[the remark this answers is not in the document]
Comment (Aditya-Rajgor) [reply examples/inlinebot.py:9]: Sure, Should I rephrase and make PR again? Or you can edit it before accepting it?

[the remark this answers is not in the document]
Comment (Bibo-Joshi) [reply examples/inlinebot.py:9]: you can just make a new commit to your master branch - the PR will automatically be updated

[the remark this answers is not in the document]
Comment (Aditya-Rajgor) [reply examples/inlinebot.py]: Updated!

**Comment by Bibo-Joshi** [comment]
Thank you for the contribution!

**Pull request merged**